package.json

```diff
@@ -8,7 +8,6 @@
     "gatsby"
   ],
   "scripts": {
-    "develop": "gatsby develop",
     "start": "gatsby develop",
```

Comment:
I prefer to keep develop & remove start. It's more explicit 🙂

Reply:
I don't know the difference these 2 scripts, but `start` generates a lot of files in the `public` folder and `develop` a few.

So, maybe, it's better to keep the both `develop` for local development & `start` for I don't know why 😆

Reply:
`npm start` is the convention used by most project to start the dev server. See also https://docs.npmjs.com/cli/v7/commands/npm-start
There is no reason we have different output for `npm start` and `npm develop` as they run the same gatsby commands.

Reply:
I have also update the README in #9 to document to use `npm start`.

Reply:
From what I saw when I run the both commands, `start` maybe run other commands because more files are generated by this command than `develop`, and I think we don't need this when we develop ^^

Reply:
ok, as this is not the purpose of this pr, let's restore the develop script

Reply:
done in 8af454e